lib/network.go

```diff
@@ -5618,7 +5618,7 @@ func (txnData *DAOCoinLimitOrderMetadata) ToBytes(preSignature bool) ([]byte, er
 	data = append(data, EncodeUint256(txnData.ScaledExchangeRateCoinsToSellPerCoinToBuy)...)
 	data = append(data, EncodeUint256(txnData.QuantityToFillInBaseUnits)...)
 	data = append(data, UintToBuf(uint64(txnData.OperationType))...)
-	data = append(data, BoolToByte(txnData.CancelExistingOrder))
+	data = append(data, txnData.CancelOrderID.ToBytes()...)
```

Comment:
If we wanted to support a nil value for `CancelOrderID` I think we can use the scheme that we use for the `PostHashToModify` in `SubmitPostMetadata.ToBytes()` function